artuslite_ros_node:
  ros__parameters:
    hand_type: "left"
    communication_method: "UART"
    communication_channel_identifier: "/dev/ttyUSB0"
    communication_frequency: 50
    wake_up: true
    calibrate: false # Set to true to calibrate the hand
    reset: false 
    feedback: false

